modules/ROOT/pages/xml-sdk.adoc

```diff
@@ -442,6 +442,22 @@ Generated schema location: `+http://www.mulesoft.org/schema/mule/hello-prefix/cu
 
 Generated schema location is `+http://www.mulesoft.org/schema/mule/hello-with-spaces/current/mule-hello-with-spaces.xsd+`
 
+[[prefix-attribute]]
+=== Prefix Attribute
```

Comment:
When I look at the preview text, it is strange to information about the `namespace` attribute in a section called "Prefix Attribute." This may have been there before you worked on this. This may be something to come back to and reword a bit.